MediaBrowser.Model/Configuration/LibraryOptions.cs

```diff
@@ -149,5 +149,19 @@ public LibraryOptions()
 
             return null;
         }
+
+        public char[] GetCustomTagDelimiters()
```

Comment:
We should stop adding buissniss logic in our DTOs.
Use a extension method instead

Reply:
The Emby time getter is kept in its old place to not break the code too much. This new one has being moved.